kyuubi-server/src/main/scala/org/apache/kyuubi/session/KyuubiBatchSessionImpl.scala

```diff
@@ -113,6 +114,17 @@ class KyuubiBatchSessionImpl(
       ms.incCount(MetricRegistry.name(CONN_OPEN, user))
     }
 
+    try {
+      checkSessionAccessPathURIs()
+      // create the operation root directory before running batch job submission operation
+      super.open()
+    } catch {
+      case e: Throwable if recoveryMetadata.isDefined =>
+        error(s"Error opening recovered batch session $handle", e)
+        Utils.tryLogNonFatalError(batchJobSubmissionOp.close())
```

Comment:
shall we close `batchJobSubmissionOp` inside `KyuubiBatchSessionImpl.close` ?

Reply:
sounds good